examples/micronaut-client/build.gradle

```diff
@@ -48,3 +42,9 @@ tasks {
     }
 }
 
+configurations.all {
```

Comment:
This is not necessary with Micronaut Framework 4.0, since Micronaut Framework 4.0 does not longer expose `micronaut-jackson-databind` automatically.